spec/lib/facilities/vha_facility_spec.rb

```diff
@@ -93,7 +94,7 @@ module Facilities
               f2_health = f2_services['health']
 
               expect(f1_services.keys).to match(%w[last_updated health other])
-              expect(f1_services['last_updated']).to eq('2019-07-09')
+              expect(f1_services['last_updated']).to eq('2017-03-31')
```

Comment:
This is a bit of a long story, I'll try to summarize. 
On the `master` branch, facilities pulls data from an arcgis endpoint that we are moving away from. The arcgis endpoint includes facility data, services data, mental health data, etc. The new gis endpoint we are using only includes facility data, not services data. On the epic branch, we updated the VCR to use data from gis but because the gis endpoint did not actually contain services data, the `last_updated` attribute was temporarily set to the date the facility data itself was updated. In this branch, the `last_updated` attribute is being set to data from Access to Care, i.e. the actual `last_updated` date for services. You can see where the old code was taken out in the removal of the `services_date` method.